app/models/storage_location.rb

```diff
@@ -162,105 +80,91 @@ def self.import_csv(csv, organization)
     end
   end
 
+  # NOTE: We should generalize this elsewhere -- Importable concern?
   def self.import_inventory(filename, org, loc)
     current_org = Organization.find(org)
     adjustment = current_org.adjustments.create(storage_location_id: loc.to_i, comment: "Starting Inventory")
-    CSV.parse(filename, headers: false) do |row|
+    # NOTE: this was originally headers: false; it may create buggy behavior
+    CSV.parse(filename, headers: true) do |row|
```

Comment:
Do we need to change the demo csv since we're using headers?

Reply:
oh yeah, maybe -- er, probably

Where is that?

Reply:
```bash
aaron@toph:~/Documents/Code/rails/diaper$ find . -name *.csv
{...}
./spec/fixtures/storage_locations.csv                      # has headers
./public/storage_locations.csv                             # has headers
{...}
aaron@toph:~/Documents/Code/rails/diaper$ 
```
We're good :+1: